src/gui/plugins/scene3d/Scene3D.hh

```diff
@@ -687,6 +706,17 @@ inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
     /// \param[in] _e The key event to process.
     public: void HandleKeyRelease(QKeyEvent *_e);
 
+    /// \brief Update the copy of entities that need an SDF geometry update
+    /// after scaling.
+    /// \param[in] _newScaledEntities The new list of entities.
+    public: void SetScaledEntities(
```

Comment:
Do you anticipate needing `SetScaledEntities` later on? The function is used once (`SetScaledEntities({})`) and I'm curious if the function should be renamed and call `scaledEntities.clear()` instead?

Reply:
Not sure yet, to be honest. I thought about it but I decided to leave it as `Set...` in case we want to change its value in the future. We can always rename it when we have the full version ready.